Lib/test/test_email/test_utils.py

```diff
@@ -75,13 +75,15 @@ def test_localtime_daylight_false_dst_false(self):
         t2 = utils.localtime(t1)
         self.assertEqual(t1, t2)
 
+    @test.support.run_with_tz('Europe/Minsk')
```

Comment:
This feels like an odd choice of test time zone, but the choice is essentially arbitrary, so I have no real problem with it.

Reply:
I guess it's copied from other tests at the bottom fo the file.

Reply:
I reused a timezone which is already used in other tests, so I don't have to pick a different timezone ;-)